wg-advisors/README.md

```diff
@@ -102,4 +109,38 @@ the ASF. Trust is transitive - that is, you may not know all members
 personally, but each member was nominated and elected by people that
 you, in turn, nominated and elected.
 
+### How to initiate PMC <-> Advisor relationship?
+
+There is no formal process for this. As an advisor, you may simply
+start participating in the project's mailing lists (dev and private)
+after introducing yourself and asking if this is fine for you to become
+an Advisor. The PMC must be allowed to say "no thank you" without providing any
```

Comment:
> become an Advisor

I don't think we'd establish a formal Advisor role in the project. But any advisor comes with a certain issue/suggestion, or shout-out (of course) for the project.

Reply:
Yeah. I added a small paragraph stating that Advisor is not the "advised PMC" role but a ComDev role.